velox/exec/Window.cpp

```diff
@@ -313,6 +459,26 @@ void Window::callApplyForPartitionRows(
     rawFrameEndBuffers.push_back(rawFrameEndBuffer);
   }
 
+  // Compute bound values for frames with k preceding and k following bounds.
+  VectorPtr precedingVector = nullptr;
+  VectorPtr followingVector = nullptr;
+  FlatVector<int>* precedingFlatVector = nullptr;
+  FlatVector<int>* followingFlatVector = nullptr;
+  vector_size_t startOffset = 0;
+  vector_size_t endOffset = 0;
+  bool hasOffsetBasedBounds = false;
```

Comment:
Are you using hasOffsetBasedBounds variable anywhere ?

Reply:
Removed it now, thanks!